hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadJdbcP.java

```diff
@@ -191,4 +204,34 @@ private static Exception close(AutoCloseable closeable) {
         }
         return null;
     }
+
+    private static void setAutoCommitIfNecessary(Connection connection, Properties properties) {
+        try {
+            String key = JdbcPropertyKeys.AUTO_COMMIT;
+            if (properties.containsKey(key)) {
+                String value = properties.getProperty(key);
+
+                if (isBoolean(value)) {
+                    boolean autoCommit = Boolean.parseBoolean(value);
+                    connection.setAutoCommit(autoCommit);
+                }
+            }
+        } catch (SQLException exception) {
+            LOGGER.severe("Error setting setAutoCommit to PreparedStatement", exception);
+        }
+    }
+
+    private static void setFetchSizeIfNecessary(PreparedStatement statement, Properties properties) {
+        try {
+            String key = JdbcPropertyKeys.FETCH_SIZE;
+            if (properties.containsKey(key)) {
+                int fetchSize = Integer.parseInt(properties.getProperty(key));
+                statement.setFetchSize(fetchSize);
+            }
+        } catch (SQLException exception) {
+            LOGGER.severe("Error setting setFetchSize to PreparedStatement", exception);
```

Comment:
IMHO we shouldn't just "eat" such exceptions

Reply:
rev : 8cdb6b481868b500b43f490407eadb9a4676e446